noetic/distribution.yaml

```
@@ -6602,6 +6602,22 @@ repositories:
       url: https://github.com/ROBOTIS-GIT/robotis_manipulator.git
       version: noetic-devel
     status: maintained
+  rom_motor_msgs:
+    doc:
+      type: git
+      url: https://github.com/GreenGhostMan/rom_motor_msgs-doc.git
+      version: main
```

```suggestion
      version: noetic-devel
```

Match the source branch.